docs/topic_guide/extending_fido.rst

```diff
@@ -27,29 +27,43 @@ A new "scraper" client inherits from `~sunpy.net.dataretriever.client.GenericCli
 * A class method :meth:`~sunpy.net.base_client.BaseClient.register_values`; this registers the "attrs" that are supported by the client.
   It returns a dictionary where keys are the supported attrs and values are lists of tuples.
   Each `tuple` contains the "attr" value and its description.
-* A class attribute ``pattern``; this is a string used to match the URLs supported by the client and extract metadata from the matched URLs.
-  The time and other metadata attributes for extraction are written using the `~sunpy.extern.parse.parse` format, using double curly-brackets so to differentiate them from ``kwargs`` parameters which are written in single curly-brackets.
+* A class attribute ``pattern``; this is a string used to match all URLs supported by the client and extract necessary metadata from the matched URLs.
+  The time and other metadata attributes for extraction are written within double curly-braces ``{{}}`` in a `parse <https://github.com/r1chardj0n3s/parse/>`__ format.
+  Regular placeholders for Python format strings can still be included via single curly braces ``{}`` with their respective parameters passed as the ``kwargs``.
+  An example of how such a pattern looks like is given in the algorithm explanation below.
+
+Note: The scraper has supported regex-based patterns for a long time, which is what it will still expect by default. However that is in the process of being replaced with parse-style patterns in the future versions of the package. To use the newer parse-style patterns, it's currently required to pass it as the ``format`` argument to the scraper.
```

Comment:
```suggestion
Note: The scraper has supported regex-based patterns for a long time, which is what it will still expect by default.
However that is in the process of being replaced with parse-style patterns in the future versions of the package.
To use the newer parse-style patterns, it's currently required to pass it as the ``format`` argument to the scraper.
```

Reply:
We want to enable the new format by default and raise a warning if its the old way.